.github/TRANSLATING.md

```diff
@@ -0,0 +1,33 @@
+# Translations (i18n)
+
+Thanks for your interest in helping to translate the documents! While we plan to support translations in the future, **we are not currently accepting them until Tauri 2.0 is a bit more stable**.
+
+Once translations are enabled you can check the [translation status page](https://beta.tauri.app/contribute/translate-status).
+
+To ensure the translation is consistent across multiple contributions, be sure to [communicate with others contributors in the `#docs` channel on Discord](https://discord.com/invite/tauri) and consult with reliable external resources such as [Microsoft terminology search](https://msit.powerbi.com/view?r=eyJrIjoiODJmYjU4Y2YtM2M0ZC00YzYxLWE1YTktNzFjYmYxNTAxNjQ0IiwidCI6IjcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0NyIsImMiOjV9) during translation.
+
+### Enabling a Language
+
+If you seek to contribute but find your language missing, please open an issue.
+
+### Starting a Translation
+
+Before you start working on a translation, look through the [open pull requests](https://beta.tauri.app/contribute/translate-status#needs-review) to see if anyone else is already working on one for your language.
+
+If none spotted, then now is time for you to lead the effort! So here's how to start:
+
+1. Head to [translation status page](https://beta.tauri.app/contribute/translate-status).
+2. Look for missing translation in corresponding languages.
+3. Hit `Create page +` which will help you automagically fork the repository and create the corresponding page.
```

Comment:
Maybe we could say "you may want to copy the contents of the English page as a starting point" or something just to be a bit more explicit? It'd be cool if there were a way for GitHub to do that automatically through that Create page button but don't think there is.

Reply:
Technically there is, same as what the current "create page +" does with the GET method:

`https://github.com/tauri-apps/tauri-docs/new/next/src/content/docs?filename=fr%2F2%2Fguide%2Findex.mdx&value=---%0Atitle%3A%0Adescription%3A%0A---%0A`

We can stuff whatever we want in that `value`, but it'll also mean some absurdly long string concatenated there which even if it works looks extremely ugly.

Let's just copy and paste for now.

Reply:
yeah, agree with you. Let's keep it simple for now to get it out the door.